Configuration/Eras/python/Era_Run2_2017_trackingLowPU_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+
+Run2_2017_trackingLowPU = cms.ModifierChain(Run2_2017, trackingLowPU)
```

Comment:
Here `Run2_2017_core` should be used instead of `Run2_2017`. Adding `trackingLowPU` will likely have strange effects as  `trackingPhase1QuadProp` is already active, and several automated constructs assume only one of the tracking sub-eras being active (hmm, maybe I should try to think how to add a check for that...). OTOH, `run2_GEM_2017` is currently only in `Run2_2017` while IMHO it should be in `Run2_2017_core`. Further OTOH, after #17605 and with #17612 we should anyway simplify the era hierarchy (I guess it's mostly up to deciding to who does the job).